apps/extension/src/routes/popup/settings/settings-clear-cache.tsx

```diff
@@ -11,15 +11,14 @@ import { SettingsScreen } from './settings-screen';
 const useCacheClear = () => {
   const navigate = usePopupNav();
   const [loading, setLoading] = useState(false);
+  const setFullSyncHeight = useStore(state => state.network.setFullSyncHeight);
 
   const handleCacheClear = () => {
     setLoading(true);
 
     void (async function () {
+      await setFullSyncHeight(undefined);
```

Comment:
issue: I don't think this will work as doing `chrome.storage.set({ "fullSyncHeight": undefined })` is a no-op. Can you confirm with a test? May help reviewing the docs as well: https://developer.chrome.com/docs/extensions/reference/api/storage#type-StorageArea.

Reply:
Truly has some problems, reverted to the Tal's initial solution with `allSettled`